tuist/tuist
# Fix openssl to use system path

Resolves https://github.com/tuist/tuist/issues/5260
Resolves https://github.com/tuist/tuist/issues/5297

### Short description 📝

If a different version of openssl takes precedence over the system bundled one it may break tuist signing features. This PR just changes all references to `openssl` to `/usr/bin/openssl`.

### How to test the changes locally 🧐

- An Xcode project that uses tuist managing signing certs and provisioning profiles is required.
- Install openssl@3 via home brew
  - `brew install openssl@3`
- Link openssl@3 via homebrew
  - `brew link openssl@3`
- Restart terminal
- Verify `which openssl` points to the homebrew install (likely `/opt/homebrew/bin/openssl`)
- Delete the existing `signing.keychain` for the project
- Clean, fetch, and generate the project with the latest release version of tuist.
- Project generation should fail with `security: SecKeychainItemImport: The specified item already exists in the keychain.`.
- Delete the existing `signing.keychain` for the project
- Fetch, and generate the project using this PRs version of tuist
- Project generation should succeed

### Contributor checklist ✅


### Reviewer checklist ✅
